python/cuml/neighbors/kneighbors_regressor.pyx

```diff
@@ -227,3 +227,8 @@ class KNeighborsRegressor(NearestNeighbors, RegressorMixin):
 
     def get_param_names(self):
         return super().get_param_names() + ["weights"]
+
+    def _more_tags(self):
+        return {
+            'preferred_input_order': 'F'
```

Comment:
Do kneighbors classifier and regressor really have different preferred orders?

Reply:
yeah, `nearest_neighbors` (the original class) calls for `F` while the reg/classifier ones call for `C` for predict :(

Reply:
Wow, yeah I think nearest neighbors was updated at one point but the classifier/regressor weren't. I'll open a PR for that one simple change.